pkg/controller/directvolumemigration/directvolumemigration_controller.go

```diff
@@ -64,7 +67,8 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		&handler.EnqueueRequestForOwner{
 			IsController: true,
 			OwnerType:    &migapi.DirectVolumeMigration{},
-		})
+		},
+		&migref.MigrationNamespacePredicate{Namespace: migapi.OpenshiftMigrationNamespace})
 	if err != nil {
```

Comment:
do you find it weird that dvm controller does not have a watch on migcluster? may be thats a bug for different time